Wabbajack/View Models/ModListVM.cs

```diff
@@ -38,6 +38,7 @@ public ModListVM(string modListPath)
             catch (Exception ex)
             {
                 Error = ex;
+                Utils.Log($"Error while loading the modlist:\n{ex}");
```

Comment:
```suggestion
                Utils.Error(ex, "Error while loading the modlist");
```
I think this would be preferable, as it gives us a concrete way of identifying which messages are "errors", vs all of them just being log lines that might be errors.. but might just be info.

If we want to eventually update the WJ log display, for ex, it would be useful to know which lines are errors, so they can be displayed red, or whatever... that sorta idea

Reply:
right, force of habit